.github/workflows/rust.yml

```diff
@@ -138,13 +138,22 @@ jobs:
         toolchain: ${{ fromJSON(needs.setup.outputs.toolchains) }}
         flags:
           - --all-features
+          - --no-default-features
     steps:
       - name: Checkout source code
         uses: actions/checkout@v2
 
       - name: Remove default members
         run: sed '/default-members/d' -i ${{ matrix.directory }}/Cargo.toml
 
+      - name: Enable default features for nng
+        run: |
+          # Search for `nng` dependencies and enable default features
```

Comment:
We've spoken off-GitHub, we want to at least make this clearer in the mean time by expanding upon the comment. We're also going to explore cargo-whatfeatures but not invest too much time right now as CI logic should probably be moved to Python or some other language at this point due to growing complexity.